Formula/skopeo.rb

```diff
@@ -16,10 +17,33 @@ class Skopeo < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "1"
+    ENV.append "CGO_FLAGS", ENV.cppflags
+    ENV.append "CGO_FLAGS", Utils.popen_read("gpgme-config --cflags")
```

Comment:
You probably want to use something like `#{Formula["gpgme"]}/bin/"gpgme-config --cflags"` here. To avoid that the formula need to be linked

Reply:
Interesting! I hadn't seen this pattern before. Done.